dbtmetabase/metabase.py

```diff
@@ -221,13 +221,21 @@ def export_model(
         else:
             model_description = model.description
 
+        body_table = {}
+        if api_table["description"] != model_description:
+            body_table["description"] = model_description
+        if api_table["points_of_interest"] != model.points_of_interest:
+            body_table["points_of_interest"] = model.points_of_interest
+        if api_table["caveats"] != model.caveats:
```

Comment:
We have a scenario here where the user has existing `caveats` defined, then removes it before the next run through. In this case api_table["caveats"] is not equal to the now Nonetype parsed caveats which will trigger an update. This update will pass the key caveats with None as the value. Metabase would definitely reject that if it were the description field so I would wager it might for the caveats / points of interest. We may need to set the dataclass up so that it doesn't allow None and default to empty str instead.

Its something we should test (verify it on a real metabase instance or in their documentation) and account for this possibility.

Reply:
Ah! True. I've basically adjusted this part, based on the `model_description` logic.